doc/howto/images_copy.md

```diff
@@ -6,3 +10,91 @@
 
   - local file
   - file on web server
+
+
+### Direct pushing of the image files
+
+This is mostly useful for air-gapped environments where images cannot be
+directly retrieved from an external server.
+
+In such a scenario, image files can be downloaded on another system using:
+
+    lxc image export ubuntu:22.04
```

Comment:
I think we could make it clearer in this section what the resulting files will be with this command and how they relate to the `import` command below.

When I ran `lxc image export ubuntu:22.04` I got:

- ubuntu-22.04-server-cloudimg-amd64-lxd.tar.xz
- ubuntu-22.04-server-cloudimg-amd64.squashfs

When I ran `lxc image export images:ubuntu/jammy` I got:

- lxd.tar.xz
- rootfs.squashfs

In both scenarios it may not be clear to the end user which file represents `META` and which represents `ROOTFS` in the command below (well maybe in the 2nd example it would be obvious that `rootfs.squashfs` is the ROOTFS, but not in the 1st example).

So perhaps we should explain that the tarball represents the metadata file and the squashfs file contains the rootfs?

Also we could call those `META` and `ROOTFS` placeholders `METADATA_FILE` and `ROOTFS_FILE` perhaps for clarity to indicate that they expect file names there?

Reply:
Also on the topic of command argument placeholders, in the rest of the docs I think we are using angle brackets to indicate the argument name right? E.g. https://linuxcontainers.org/lxd/docs/master/howto/instances_access_files/#pull-files-from-the-instance-to-the-local-machine

> `lxc file pull <instance_name>/<path_to_file> <local_file_path>`

So I think this should be something like:

```
lxc image import <metadata_file_path> <rootfs_file_path> --alias ubuntu-22.04
```